docs/source/quickstart-pt-2.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: "Federation quickstart (preview)"
+sidebar_title: "Part 2 - Composition in Apollo Studio"
+description: "Part 2 - Composition in Apollo Studio"
+---
+
+import {Button} from '@apollo/space-kit/Button';
+import {Link} from 'gatsby';
+import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
+
+> **Reminder:** This tutorial uses some Apollo tools and terminology that are still in **preview**, and therefore this content is not yet recommended for use in a production environment.
+
+[Back in Part 1](./quickstart/), we created a federated gateway and used the Rover CLI to compose a supergraph schema for it based on two subgraphs. Next, we'll move that composition process into the cloud.
+
+Our gateway currently loads its composed supergraph schema from a file on startup. This means that whenever the schema _changes_, the gateway needs to restart to load the new version.
+
+We can avoid this downtime by instead composing our supergraph schema with **managed federation**, a free cloud-based service that's part of Apollo Studio.
+
+## Managed federation basics
+
+With managed federation, each of our subgraphs registers its schema with Apollo. Whenever a subgraph schema changes, Apollo composes a new supergraph schema.
+
+```mermaid
+flowchart BT;
+  subgraph " "
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  end  
+  registry{{Apollo Schema Registry}};
+  serviceA & serviceB -. Registers schema .-> registry;
+  registry -- Composes supergraph schema--> registry;
+  class registry secondary;
+```
+
+Meanwhile, our gateway regularly polls Apollo for changes to its supergraph schema:
```

Comment:
@jstjoe would it make sense to introduce that supergraphs are published to an `Uplink` and that is what the gateway will poll? Think we'll need to introduce the concept of an uplink since the gateway  requires the `APOLLO_SCHEMA_CONFIG_DELIVERY_ENDPOINT=https://uplink.api.apollographql.com/` to be set as shown in the [managed federation section of the supergraph-demo](https://github.com/apollographql/supergraph-demo#managed-federation)

Reply:
Yep that makes sense. 👍